packaging/opensuse/snapd.spec

```diff
@@ -15,6 +16,22 @@
 
 %bcond_with testkeys
 
+# Enable AppArmor on openSUSE Tumbleweed (post 15.0) or higher
+%if 0%{?suse_version} >= 1550
+%bcond_without apparmor
+%else
+%bcond_with apparmor
+%endif
+
+# Compat macros
+%{!?make_build: %define make_build %{__make} %{?_smp_mflags}}
```

Comment:
This does not seem to work on Leap 42.3. At least `%make_build` appears to not be expanded in `%build` and `%check`. Can't we just define the macro here instead of relying on conditional expansion?

Reply:
Changed it to a global define, but it should not be defined by the spec if it already exists...